lab1 init

wild-boomerang opened the pull request:
Lab 1 is ready, I guess

wild-boomerang committed:
lab1 init

gmltA commented on Lab_1/app/src/paid/java/by/bsuir/wildboom/lab1/UnitFragment.java:
```diff
@@ -0,0 +1,162 @@
+
+import androidx.lifecycle.ViewModelProvider;
+
+public class UnitFragment extends Fragment {
```
Remove code duplication by extending UnitFragment from the free version here.

wild-boomerang replied:
I removed code duplication by adding UnitFragmentBase abstract class in main version. In free and paid versions each UnitFragment extends UnitFragmentBase. I did this because I didn't know how a class from the paid version could extend a class from the free version.

wild-boomerang committed:
code duplication was removed

now UnitFragment from free and paid version extend abstract class UnitFragmentBase from main

gmltA reviewed (approved):
That's better.
